src/network/bridge.rs

```diff
@@ -133,9 +143,11 @@ impl driver::NetworkDriver for Bridge<'_> {
             data.bridge_interface_name, data.ipam.gateway_addresses
         );
 
-        setup_ipv4_fw_sysctl()?;
-        if data.ipam.ipv6_enabled {
-            setup_ipv6_fw_sysctl()?;
+        if let BridgeMode::L3 = data.mode {
```

Comment:
If podman passed an unsupported mode it would fail in `get_bridge_mode_from_string`. In case you mean "in case we add support for the new mode, but forget to handle it in all places", we need to use `match` everywhere. It might make some conditions more complicated but it would cause compile-errors.

Reply:
yes I meant we forget to handle it. However if we go unmanaged/managed for now I think it is safe enough that we do not add new modes often enough. And if we do  we need to check what mode should do what anyway so let's keep the readable form.

Reply:
makes sense. 

> Also now that I look at this we should really not set the global sysctl for internal networks here (feel free to fix this in a second commit or I put it on my list)

done